src/components/Composer/index.js

```diff
@@ -306,11 +306,7 @@ class Composer extends React.Component {
      * @param {ClipboardEvent} event
      */
     handlePaste(event) {
-        if (!this.props.checkComposerVisibility()) {
-            return;
-        }
-
-        if (['INPUT', 'TEXTAREA'].includes(event.target.nodeName)) {
+        if (!this.props.checkComposerVisibility() && !this.state.isFocused) {
```

Comment:
This does not fix anything

Reply:
Need to add this check also ` if (this.textInput.id !== event.target.id) { return; }`

Reply:
Are you suggesting we keep `if (!this.props.checkComposerVisibility()) {` then? I went with a straight revert but I can update if you'd prefer.

Reply:
Yeah, this bug [here](https://github.com/Expensify/App/pull/22817/files#r1263034541) seems to be low on priority and better than both!

Reply:
Updated

Reply:
So we could create a new issue for that alone while we look for a solution. What do you say?

Reply:
Sorry we must have posted at the same time and GitHub didn't initially show me [this message](https://github.com/Expensify/App/pull/22845#discussion_r1263057149).

We're discussing 1:1 now to figure out the best solution.

Reply:
Update for visibility: we looked into a few other options but didn't find one that fully worked. The current plan is to:

- Go ahead and merge this revert PR so that we can unblock the production deploy (we're kind of in a rush today so we can deploy other important changes)
- Look for a proper solution later on